### DISCUSSION TITLE
dependabot : useful ?

### DISCUSSION THREAD

**Discussion by sdesrozis** [pytorch/ignite#1886] [Q&A]
## 🚀 Feature

`dependabot` Is a tool to automatically update dependencies of projects https://dependabot.com/

It is natively into GitHub https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/

Do you think it could help ?

**Comment by vfdev-5** [comment]
@sdesrozis I transferred this to Q&A discussions.
